css/discussion.css

```diff
@@ -204,20 +203,30 @@ div#TB_ajaxWindowTitle {
 }
 .gp-content ul.feedback-reason-list {
 	list-style-type: none;
-	display: inline-block;
-	padding-left: .5em;
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-column-gap: 4px;
+    margin-left: 0;
 }
 .feedback-reason-list li {
-	width: 50%;
 	float: left;
 	padding-top: 4px;
 }
 .feedback-reason-list li label {
 	font-size: 1.1em;
 	cursor: pointer;
+	min-width: 100%;
 }
 .feedback-reason-list li input[type="checkbox"] {
-	margin-right: 5px;
+	float: left;
+}
+.feedback-reason-list li span.gp-reason-text {
+	display: inline-block;
+    margin-bottom: 5px;
+    float: left;
+    width: 70%;
+    margin-left: 2px;
+    margin-top: -8px;
```

Comment:
There is mixed whitespace, looks like you had your editor misconfigured to use spaces instead of tabs? Could you please update the whole file?

Also I recommend to use relative units like em where possible. Here the text was too close to the checkbox for my taste. This is what my suggested CSS looks like:
![Screenshot 2023-02-27 at 08 53 19](https://user-images.githubusercontent.com/203408/221505575-8d14d3f9-e707-493f-ac02-18bc1e6be4b3.png)


```suggestion
	display: inline-block;
	margin-bottom: .5em;
	float: left;
	width: 70%;
	margin-left: .5em;
	margin-top: -.5em;
```